src/test/java/picard/util/GCloudTestUtils.java

```diff
@@ -0,0 +1,61 @@
+package picard.util;
+
+public final class GCloudTestUtils {
+    /**
+     *  This is a public requester pays bucket owned by the broad-gatk-test project.
+     *  It must be owned by a different project than the service account doing the testing or the test may fail because it can access the
+     *  file directly through alternative permissions.
+     */
+    public static final String REQUESTER_PAYS_BUCKET_DEFAULT = "gs://hellbender-requester-pays-test/";
+
+    public static final String TEST_INPUTS_DEFAULT = "gs://hellbender/test/resources/";
+    public static final String TEST_STAGING_DEFAULT = "gs://hellbender-test-logs/staging/";
```

Comment:
It should not cause issues unless we decide to blow up the hellbender ones at some point in the future.  It seemed like the sanest thing to do since the various permissions and buckets are configured correctly and I'm not 100% sure I remember what all the settings had to be.